.cases/kafka-create.md

```diff
@@ -0,0 +1,19 @@
+## Kafka provider verification
+
+## Requirements
+
+ - Create new account 
+
+## Cases
+
+rhoas command
+> output
+
+rhoas kafka create --name=wtrocki --provider=azure
+❌ azure is not a valid or currently enabled cloud provider name. Valid names are: aws 
```

Comment:
```suggestion
❌ "azure" is not a valid or currently enabled cloud provider name. Valid names are: aws 
```

I think that we should quote the value here to make it clearer that this is the value entered.